README.md

```diff
@@ -125,3 +125,7 @@ Here is a list of the more detailed features including a link to the documentati
  * scales from small indoor-sized to world-wide-sized graphs
  * Find nearest point on street e.g. to get elevation or 'snapp to road'
  * Enables the [map matching component](https://github.com/graphhopper/map-matching) (flexibility only)
+
+# Attribution
```

Comment:
Maybe we should link to a separate Readme inside the gtfs reader module? As I fear there will be more than one of these attribution statements necessary and it is not really related to all GH users e.g. the once not using public transport. Or use a different test DB where this is not necessary?